doc/source/serve/key-concepts.md

```diff
@@ -30,6 +30,14 @@ handle = serve.run(my_first_deployment)
 print(ray.get(handle.remote())) # "Hello world!"
 ```
 
+(serve-key-concepts-application)=
+
+## Application
+
+An application is the unit of upgrade in a Ray Serve cluster. An application consists of one or more deployments. One of these deployments is considered the “ingress” deployment, which handles all inbound traffic.
```

Comment:
can we add ingress deployment as a concept and link it?

Reply:
It's on the key concepts page already, I can do an in-page link to it.

Reply:
perfect